packages/typespec-client-generator-core/test/rules/require-client-suffix.test.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  BasicTestRunner,
+  createLinterRuleTester,
+  LinterRuleTester,
+} from "@typespec/compiler/testing";
+import { beforeEach, describe, it } from "vitest";
+import { requireClientSuffixRule } from "../../src/rules/require-client-suffix.rule.js";
+import { createSdkTestRunner } from "../test-host.js";
+
+describe("typespec-client-generator-core: require-client-suffix", () => {
```

Comment:
nit: no need to add those top level describe with vitest, they just add extra nesting in the UI